testing/test_column_line_numbers.py

```diff
@@ -0,0 +1,57 @@
+import ast
+from pathlib import Path
+from typing import List, Tuple
+
+import pytest
+from flake8_annotations import checker
+
+
+TEST_FILE = Path("./testing/code/column_line_numbers.py")
+
+# (line, column) tuples where we should get linting errors in the test file
+# Line numbers are 1-indexed
+# Column offsets are 0-indexed when yielded by our checker; flake8 adds 1 when emitted
+SHOULD_ERROR = ((16, 8), (16, 11), (24, 4), (25, 4), (26, 2))
+ERROR_CODE = Tuple[int, int, str, checker.TypeHintChecker]
+
+
+@pytest.fixture
+def parsed_errors(src_filepath: Path = TEST_FILE) -> List[ERROR_CODE]:
+    """Create a fixture for the error codes emitted by our testing code."""
+    with src_filepath.open("r") as f:
+        src = f.read()
+
+    tree = ast.parse(src)
+    lines = src.splitlines()
+
+    return list(checker.TypeHintChecker(tree, lines).run())
```

Comment:
Why isn't this left as a generator? Does pytest use the same instance of a fixture for all tests?

Reply:
I don't know. I think it's an artifact from an earlier (fixtureless) approach that I never changed.